src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import '@polkadot/api-augment'
+import '@kiltprotocol/augment-api'
```

Comment:
`@polkadot/api-augment` vs `@kiltprotocol/augment-api` did we swap the name here and broke the convention? 😂

Reply:
was there a convention? 😨

Reply:
Not a convention. but if polkadot names it `api-augment` i would consider it the established name.

Reply:
welllllll too late